<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainSettings</class>
 <widget class="QWidget" name="TrainSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Train Settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QListWidget" name="listWidgetTrains">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Train Details</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelName">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelProtocol">
          <property name="text">
           <string>Protocol:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxProtocol">
          <item>
           <property name="text">
            <string>Analog</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Märklin MM1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Märklin MM2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NMRA/DCC 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NMRA/DCC 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>By Server</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelDecoderSpeedSteps">
          <property name="text">
           <string>Decoder Speed Steps:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBoxSpeedSteps"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelDecoderFunctions">
          <property name="text">
           <string>Decoder Functions:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBoxDecoderFunctions"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelAddress">
          <property name="text">
           <string>Address:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spinBoxAddress"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonNew">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRemove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
